DayTomato/DayTomato.Droid/Fragments/MapFragment.cs

```diff
@@ -168,7 +183,15 @@ private void SetListeners()
 		public async void OnDialogClosed(object sender, DialogEventArgs e)
 		{
 			Account account = await MainActivity.GetUserAccount();
-			Pin pin = new Pin(0, e.Name, e.Description, 0, _selectLocation.Latitude, _selectLocation.Longitude, account.Id);
+			Pin pin = new Pin(0,
```

Comment:
C# has a nice initialization syntax for classes whereby you can do
```C#
new Pin 
{
  Type = 0,
  Name = e.Name,
  Description = e.Description,
 ...
}
```

Saves from trying to remember the order of parameters!